src/engine/app.rs

```diff
@@ -64,10 +65,11 @@ impl Application {
         assets.add_loader::<gfx_types::Factory>(factory);
 
         let trans_sys = TransformSystem::new();
-        planner.add_system::<TransformSystem>(trans_sys, "transform_system", 0);
+
+        let dispatcher_builder = dispatcher_builder.add_barrier();
```

Comment:
As discussed on Gitter, it would be preferable to give the responsibility of adding "default" systems to the user.